gammapy/utils/tests/test_time.py

```diff
@@ -0,0 +1,27 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from numpy.testing import assert_almost_equal
+from astropy.time import Time, TimeDelta
+from ..time import time_ref_from_dict, time_relative_to_ref
+
+
+def test_time_ref_from_dict():
+    mjd_int = 500
+    mjd_frac = 0.5
+    time_ref_dict = dict(MJDREFI=mjd_int, MJDREFF=mjd_frac)
+    time_ref = time_ref_from_dict(time_ref_dict)
+    decimal = 4
```

Comment:
This temp variable doesn't help, you can save one line and just pass this as a keyword argument in the next line:
```python
assert_almost_equal(time_ref.mjd, mjd_int + mjd_frac, decimal=4)
```